docs/howtos/cut-a-new-release.md

```diff
@@ -1,17 +1,20 @@
 # Application Services Release Process
 
 ## Make a new release from latest main.
+1. Validate your changes for consumers.
+    - Smoketest your changes in iOS, android-components, and fenix to ensure the release will integrate cleanly with key downstream consumers
+      - Run the `./automation/smoke-test-firefox-ios.py` script to test integration with Firefox for iOS.
+      - Run the `./automation/smoke-test-android-components.py` script to test integration with Android Components.
+      - Run the `./automation/smoke-test-fenix.py` script to test integration with Fenix.
 
-1. Smoketest whether the release will integrate cleanly with key downstream consumers:
-    - Run the `./automation/smoke-test-firefox-ios.py` script to test integration with Firefox for iOS.
-    - Run the `./automation/smoke-test-android-components.py` script to test integration with Android Components.
-    - Run the `./automation/smoke-test-fenix.py` script to test integration with Fenix
-    - Build and run the [sync sample app](https://github.com/mozilla-mobile/android-components/tree/master/samples/sync) in android-components
-      against your local checkout using the [local publishing flow](./locally-published-components-in-fenix.md).
-
-    If these tests fail, it indicates a breaking change. Check that such a change is called out
+      **Note:** If these tests fail, it indicates a breaking change. Check that such a change is called out
     in the changelog, and file downstream bugs to track the work of integrating it.
-2. Run the `./automation/prepare-release.py` script to create a release commit and open a pull-request. If this script does not work for you, please refer to the "Create a release commit manually" section in this document.
+    - Run the android-components [sync sample app](https://github.com/mozilla-mobile/android-components/tree/master/samples/sync) to ensure sync functionality hasn't been adversely impacted using the [local publishing flow](./locally-published-components-in-fenix.md).
+
+2. Create a release commit.
+    - **Automated process:** run the `./automation/prepare-release.py` script to create a release commit and open a pull-request.
```

Comment:
I was confused at this step previously.  

- We don't mention updating the changelog file from unreleased changes.
- Does the PR have to be reviewed and merged and ci completed before moving to step 3?

Reply:
The script should have been updating the changelog file. I've created #4138 to hopefully address this.